Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-volume-clone.md

```diff
@@ -17,8 +17,12 @@ for more info.
 
 In [pvc-clone](https://github.com/rook/rook/tree/master/deploy/examples/csi/rbd/pvc-clone.yaml),
 `dataSource` should be the name of the `PVC` which is already created by RBD
-CSI driver. The `dataSource` kind should be the `PersistentVolumeClaim` and also storageclass
-should be same as the source `PVC`.
+CSI driver. The `dataSource` kind should be the `PersistentVolumeClaim`.
+The `storageClassName` can be any storageclass (not necessarily same as Parent PVC) with the
```

Comment:
!!! note

Reply:
The `storageClassName` can be any RBD storageclass (not necessarily same as Parent PVC) with the